src/util/is.ts

```diff
@@ -0,0 +1,13 @@
+import { GenericObject } from '../types';
```

Comment:
You can remove this reference to `GenericObject` if you use proper types for `nodeType` and `setInterval`.